.github/CODEOWNERS

```diff
@@ -42,7 +42,7 @@
 /sdk/containerregistry/ @jeremymeng
 
 # PRLabel: %Cosmos
-/sdk/cosmosdb/ @southpolesteve @zfoster
+/sdk/cosmosdb/ @kushagraThapar
```

Comment:
@ramya-rao-a - let's add @simorenoh, @simplynaveen20, @xinlian12 and @moderakh to start with. I will add more folks if needed. Thank you :)

Reply:
Sure, done in https://github.com/Azure/azure-sdk-for-js/pull/17433/commits/34e834a6db2bd4991e5e48196b3a946a73189ad4